src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -76,19 +81,30 @@ public sealed class PolicyContract : NativeContract
         private const byte Prefix_ExecFeeFactor = 18;
         private const byte Prefix_StoragePrice = 19;
         private const byte Prefix_AttributeFee = 20;
+        private const byte Prefix_MSPerBlock = 21;
         private const byte Prefix_MaxValidUntilBlockIncrement = 22;
 
         private readonly StorageKey _feePerByte;
         private readonly StorageKey _execFeeFactor;
         private readonly StorageKey _storagePrice;
+        private readonly StorageKey _msPerBlock;
         private readonly StorageKey _maxValidUntilBlockIncrement;
 
+        /// <summary>
+        /// The event name for the block generation time changed.
+        /// </summary>
+        private const string MSPerBlockChangedEventName = "MSPerBlockChanged";
```

Comment:
Also change here, "TimePerBlockChanged" or even better "ConsensusTimePerBlockChanged"

Reply:
This name must be kept in accordance with the getter/setter name, so we firstly need to agree on https://github.com/neo-project/neo/pull/3895#discussion_r2041711348.